Automattic/wp-calypso
# Stats: DateRange: clears out old datepicker code

## Proposed Changes

* This PR removes the old code, and feature flag, from the previous datepicker implementation

## Why are these changes being made?


* Part of the date control update project

## Testing Instructions



* load the Calypso live branch or
* navigate to the traffic page at `/stats/day/{site URL}`
* open up the daterange picker, and check that everything still all works as expected, and functionality is maintained

## Pre-merge Checklist
